kapitan/cached.py

```diff
@@ -7,8 +7,10 @@
 
 "cached module"
 from argparse import Namespace
+from kadet import Dict
 
 inv = {}
+global_inv = Dict({})
```

Comment:
I don't see any advantages of making this a `kadet.Dict` globally, but I might be missing something. This definitely hurts performance compared to having it as a regular `dict`.

```suggestion
global_inv = {}
```